### ISSUE TITLE
No way to disable the callback set_mavlink_callback

### ISSUE THREAD

**Issue by hamishwillee** [dronekit/dronekit-python#115]
We can add a callback (though it is not recommended) using set_mavlink_callback but there is no way to remove it.

**Comment by tcr3dr** [comment]
There's no way simple way to test that this issue causes wrong behavior except by looking at the verbose logs, as indeed it silently fails! But I've added a test that checks for regressions.

The way to "unset" set_mavlink_callback as of #210 is:

```
v.set_mavlink_callback(None)
```

**Comment by hamishwillee** [comment]
Thanks @tcr3dr . I will update the documentation when #210 is merged.

**Comment by hamishwillee** [comment]
TODO Update docs when #225 is merged (supersedes #210)

**Comment by tcr3dr** [comment]
@hamishwillee #240 has been merged which fixes this issue. Documentation was added for `unset_mavlink_callback` but it may need to be improved. Let me know.

**Comment by hamishwillee** [comment]
@tcr3dr I've added #245 to improve the documentation. What you had was fine (though you needed a space after the ``..code: python`` line to get the code block to render). The changes add information about the API to the guide and example code. I took the opportunity to improve the related docs.

In addition, I couldn't merge this directly onto the main repo (had to do it via my fork because I got a 403). Hope the approach is OK. 

Yes, this can be closed when minor branch is merged.
